tensorflow/lite/tools/optimize/quantize_weights.cc

```diff
@@ -52,12 +52,12 @@ std::vector<ConsumerOpInfo> GetTensorConsumers(const ModelT* model,
   // TODO(suharshs): If this proves to be too slow, avoid calling it per tensor,
   // instead doing one sweep for the entire model.
   std::vector<ConsumerOpInfo> consumer_ops;
-  for (int op_idx = 0; op_idx < subgraph->operators.size(); ++op_idx) {
+  for (size_t op_idx = 0; op_idx < subgraph->operators.size(); ++op_idx) {
     OperatorT* op = subgraph->operators[op_idx].get();
     if (op == nullptr) {
       continue;
     }
-    for (int i = 0; i < op->inputs.size(); ++i) {
+    for (size_t i = 0; i < op->inputs.size(); ++i) {
       if (op->inputs[i] == tensor_idx) {
         consumer_ops.push_back({op, op_idx, i});
```

Comment:
We need to `static_cast<int>` `op_idx` & `i` here, since the initializer expects integers.

Reply:
@srjoglekar246 , thanks for your review comments, i have updated the code, kindly check and approve.